packages/neuron-wallet/src/block-sync-renderer/sync/queue.ts

```diff
@@ -47,9 +50,12 @@ export default class Queue {
   start = async () => {
     logger.info("Queue:\tstart")
     try {
-      this.#indexerConnector = new IndexerConnector(this.#addresses, this.#url, this.#indexerUrl)
-
-      await this.#indexerConnector.connect()
+      if (this.#url === 'http://localhost:9000') {
```

Comment:
The url could be replaced by https://github.com/nervosnetwork/neuron/pull/2590/files#diff-486b3b6eb77724b4fb3b13a4b64f038277a6a98d78a62d12959ef22a3dd5bbb6R4